spyder/plugins/completion/languageserver/confpage.py

```diff
@@ -692,17 +692,59 @@ class LanguageServerConfigPage(GeneralConfigPage):
     def setup_page(self):
         newcb = self.create_checkbox
 
-        # --- Introspection ---
-        # Basic features group
-        basic_features_group = QGroupBox(_("Basic features"))
-        completion_box = newcb(_("Enable code completion"), 'code_completion')
+        # --- Completion ---
+        # Completion group
+        completion_label = QLabel(_('Some text for this section!'))
+        self.completion_box = newcb(_("Enable code completion"),
+                                    'code_completion')
+        self.completion_hint_box = newcb(
+            _("Show completion details"),
+            'completions_hint',
+            section='editor')
+        self.automatic_completion_box = newcb(
+            _("Show completions on the fly"),
+            'automatic_completions',
+            section='editor')
+        self.completions_after_characters = self.create_spinbox(
+            _("Show automatic completions after characters:"), None,
```

Comment:
Let's follow RStudio here and change this text to be

    Show automatic completions after characters entered: